tests/test_self.py

```diff
@@ -96,42 +100,53 @@ def _configure_lc_ctype(lc_ctype):
 
 
 class MultiReporter(BaseReporter):
-    def __init__(self, reporters):
+    def __init__(
+        self,
+        reporters: List[
+            Union[JSONReporter, ColorizedTextReporter, TextReporter, BaseReporter]
```

Comment:
```suggestion
            Reporter
```
(Not sure about that but the base class would be clearer imo.)